docs/content/getting-started/index.md

```diff
@@ -87,14 +87,21 @@ This command:
 
 - Runs the application in your Kubernetes cluster
 - Creates a port-forward from localhost to port 3000 inside the container so you can navigate to the app's frontend UI
+- Creates a port-forward from localhost to port 7007 inside the container so you can navigate to your Radius Dashboard
 - Streams container logs to your terminal
 
+<br>
 In your browser you should see the demo app:
 
 {{< image src="demo-screenshot.png" alt="Screenshot of the demo container" width=600px >}}
-<br /><br />
+<br>
+
+Access your Radius Dashboard by opening [http://localhost:7007](http://localhost:7007) in a browser. In your browser, you should see the Radius Dashboard, which includes visualizations of the application graph, environments, and recipes:
```

Comment:
For the getting started guide, the environment name and application name are well known. Can we deep-link directly into the application graph page to drop the user directly into the application graph, so that they instantly go from authoring their app to seeing their app without needing to pause and navigate the UI?